internal/mode/static/state/graph/route_common.go

```diff
@@ -870,166 +881,6 @@ func validateHeaderMatch(
 	return allErrs
 }
 
-func validateFilterHeaderModifier(
```

Comment:
Functions for filters that are common between GRPC and HTTP routes have been moved to `common_filter.go`